src/styles/components/_our-goal-box.scss

```diff
@@ -0,0 +1,48 @@
+@import '../abstracts/_abstracts-dir';
+
+.our-goal-box {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border: solid 1px $silver-chalice;
+
```

Comment:
What about adding the `padding-top/bottom` at the block level instead of adding at `h` and `p`?